linux_os/guide/system/var_nonlocal_paths.var

```diff
@@ -0,0 +1,16 @@
+documentation_complete: true
+
+title: 'Explicit non-local paths'
+
+description: |-
+    Colon-separated list of paths that are not local and shouldn't be scanned if the OVAL check specifies <code>recurse_file_system="local"</code>
```

Comment:
I wouldn't mention `recurse_file_system=local`. That's a different mechanism which is performed at different level. What your OVAL does is filtering based on states. I would describe it this way:
Colon-separated list of paths that are not local and that will be filtered out and will not be scanned.